src/util/coordinateconversion.cpp

```diff
@@ -24,26 +24,301 @@
+    }
+
+    bool isRaDecValid(int raH, int raM, double raS, int decD,
+                      int decM, double decS)
+    {
+        // Ra
+        if (raH < 0.0 || raH >= 24.0) {
+            LWARNING(fmt::format("Right ascension hours '{}' is outside the allowed "
+                "range of 0 to 24 hours (exclusive)", raH)
+            );
+            return false;
+        }
+        if (raM < 0.0 || raM >= 60.0) {
+            LWARNING(fmt::format("Right ascension minutes '{}' is outside the allowed "
+                "range of 0 to 60 minutes (exclusive)", raM)
+            );
+            return false;
+        }
+        if (raS < 0.0 || raS >= 60.0) {
+            LWARNING(fmt::format("Right ascension seconds '{}' is outside the allowed "
+                "range of 0 to 60 seconds (exclusive)", raS)
+            );
+            return false;
+        }
+
+        // Dec
+        if (decD < -90.0 || decD > 90.0) {
+            LWARNING(fmt::format("Declination degrees '{}' is outside the allowed range "
+                "of -90 to 90 degrees (inclusive)", decD)
+            );
+            return false;
+        }
+        else if ((decD == -90.0 || decD == 90.0) && (decM != 0 || decS != 0)) {
+            LWARNING("Total declination is outside the allowed range of -90 to 90 "
+                "degrees (inclusive)"
+            );
+            return false;
+        }
+        if (decM < 0.0 || decM >= 60.0) {
+            LWARNING(fmt::format("Declination minutes '{}' is outside the allowed range "
+                "of 0 to 60 minutes (exclusive)", decM)
+            );
+            return false;
+        }
+        if (decS < 0.0 || decS >= 60.0) {
+            LWARNING(fmt::format("Declination seconds '{}' is outside the allowed range "
+                "of 0 to 60 seconds (exclusive)", decS)
+            );
+            return false;
+        }
+
+        return true;
+    }
+} // namespace
+
 namespace openspace {
 
-glm::dvec3 icrsToGalacticCartesian(float ra, float dec, double distance) {
-    // Convert to Galactic Coordinates from ICRS right ascension and declination
-    // https://gea.esac.esa.int/archive/documentation/GDR2/Data_processing/
-    // chap_cu3ast/sec_cu3ast_intro/ssec_cu3ast_intro_tansforms.html#SSS1
-    const glm::dmat3 conversionMatrix = glm::dmat3({
-        -0.0548755604162154,  0.4941094278755837, -0.8676661490190047, // col 0
-        -0.8734370902348850, -0.4448296299600112, -0.1980763734312015, // col 1
-        -0.4838350155487132,  0.7469822444972189,  0.4559837761750669  // col 2
-    });
-
-    glm::dvec3 rICRS = glm::dvec3(
-        cos(glm::radians(ra)) * cos(glm::radians(dec)),
-        sin(glm::radians(ra)) * cos(glm::radians(dec)),
-        sin(glm::radians(dec))
+// Convert Equatorial coordinates ICRS right ascension and declination (a, d)
+// into Galactic coordinates (l, b)
+glm::dvec3 icrsToGalacticCartesian(double ra, double dec, double distance)
+{
```

Comment:
This `{` looks like it can be moved to the previous line and still keep a line width < 90